### PR TITLE
pdctl: add label command

### PR DESCRIPTION
**Pull request by nolouch**
close https://github.com/pingcap/pd/issues/505 https://github.com/pingcap/pd/issues/507
PTAL @siddontang @huachaohuang @overvenus

### CODE REVIEW

File: pdctl/command/global.go [hunk 1]
```diff
@@ -26,7 +27,59 @@ import (
...
+func validPDAddr(pd string) error {
+	u, err := url.Parse(pd)
+	if err != nil {
+		return err
+	}
+	if u.Scheme == "" {
+		u.Scheme = "http"
+	}
+	addr := u.String()
+	reps, err := http.Get(fmt.Sprintf("%s/pd/ping", addr))
```
Comment (overvenus) [inline pdctl/command/global.go]: Why not member API?

File: pdctl/command/global.go [hunk 2]
```diff
@@ -26,7 +27,59 @@ import (
...
+	if err != nil || reps.StatusCode != http.StatusOK {
```
Comment (siddontang) [inline pdctl/command/global.go]: handle err first, then handle status code.

**Comment by huachaohuang** [comment]
You should split into two PR, one for refactor, another for the label command.

**Comment by siddontang** [comment]
Any update @nolouch ?

**Comment by nolouch** [comment]
i will split it

**Pull request closed without merging**